### PR TITLE
fix(bug): Make the Hai Start actually show up

### PR DESCRIPTION
**Pull request by TheGiraffe3**
**Bug Fix**

This PR addresses the bug described on [Discord](https://discord.com/channels/251118043411775489/1031714960528646194/1300452901151576104)

## Summary
Added a missing `global` in front of the `to reveal` and `to unlock` conditions in the Hai Start.
Also added a missing copyright line for Hurleveur in the `starts.txt` file.
@Arachi-Lover would you test this PR and see if it works?

### CODE REVIEW

**Review by Anarchist2 — COMMENTED** [review]
[Starts already remove `"global: "` when evaluating their conditions.](https://github.com/endless-sky/endless-sky/wiki/Creating-Starts#conditional-starts) The reason the start doesn't show up is because the global condition set is `"Ask the Hai about the Unfettered"`, not `"Ask the Hai about the Unfettered: offered"`.

**Comment by TheGiraffe3** [comment]
> [Starts already remove `"global: "` when evaluating their conditions.](https://github.com/endless-sky/endless-sky/wiki/Creating-Starts#conditional-starts) The reason the start doesn't show up is because the global condition set is `"Ask the Hai about the Unfettered"`, not `"Ask the Hai about the Unfettered: offered"`.

That makes sense. Thanks for the clarification!

**Comment by Anarchist2** [comment]
The numbers for date/credits/debt also need to be converted into their final formatting. Credit values and dates under an `on reveal` node don't have any formatting applied to them, so they'll need to be converted accordingly (`"Tue, 16 Nov 3013"` and `1.431M` respectively).

**Comment by TheGiraffe3** [comment]
Are you sure about that?
What if somebody changes the date format?

**Comment by TheGiraffe3** [comment]
Does anybody know why checks haven't started yet?
It's been about forty-five minutes since the last commit.

**Comment by tibetiroka** [comment]
They seem to have all completed.

**Comment by TheGiraffe3** [comment]
They have now.

File: data/starts.txt [hunk 1]
```diff
@@ -67,19 +68,19 @@ start "hai space"
...
 	on reveal
 		name "Hai Origins"
 		description `You grew up on Greenwater, a Hai world with a rich and mixed culture thanks to its climate. You've dreamed of owning your own fleet since you were given toy spaceships as a small child. To reach that goal, you worked hard at a travel company, recommending locations in human space.`
 		description `	Now, as your ferry docks at the main spaceport, you feel like this is the start of a new life as a captain.`
 		system "Fah Soom"
 		planet "Greenwater"
-		date 16 11 3013
-		credits 1431000
-		debt 1431000
+		date Tue, 16 Nov 3013
```
Comment (warp-core) [inline data/starts.txt:78]: When this is loaded, it simply puts the token after "date" into a string.
What you have here will result in the "date" that is displayed when the start is in this state being "Tue,".
```suggestion
		date "Tue, 16 Nov 3013"
```

**Comment by warp-core** [comment]
> What if somebody changes the date format?

Only the "unlocked" state supports that. The other states just put whatever is in the data into a string.

File: data/starts.txt [hunk 2]
```diff
@@ -67,19 +68,19 @@ start "hai space"
...
 		description `You grew up on Greenwater, a Hai world with a rich and mixed culture thanks to its climate. You've dreamed of owning your own fleet since you were given toy spaceships as a small child. To reach that goal, you worked hard at a travel company, recommending locations in human space.`
 		description `	Now, as your ferry docks at the main spaceport, you feel like this is the start of a new life as a captain.`
 		system "Fah Soom"
 		planet "Greenwater"
-		date 16 11 3013
-		credits 1431000
-		debt 1431000
+		date "Tue, 16 Nov 3013"
+		credits 1.431
+		debt 1.431
```
Comment (warp-core) [inline data/starts.txt:80]:
```suggestion
		credits "1.431M"
		debt "1.431M"
```

**Pull request merged**